Changelog.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+<!---
+SPDX-License-Identifier: 2.0 license with LLVM exceptions
+--->
+
+## [Unreleased]
+
+### Added
+
+- [LIBRARY_STATUS]: Library status updated to [Under development and not yet ready for production use.](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_LIBRARY_MATURITY_MODEL.md#under-development-and-not-yet-ready-for-production-use): It is not yet ready for production use.
```

Comment:
I have questions that occur to me seeing this. They won't affect the review status on this PR itself.

* Will the Exemplar always be in an "under development" status since it doesn't have a numbered paper?
* Should we elaborate on why it's "under development" and not another more mature status?
* Do we want to release it to package managers anyway to demonstrate and test those workflows on behalf of the rest of the Beman libraries?

Reply:
> Will the Exemplar always be in an "under development" status since it doesn't have a numbered paper?
Yes, IMO. 

> Will the Exemplar always be in an "under development" status since it doesn't have a numbered paper?
Good suggestion, pushed that into a commit.

> Do we want to release it to package managers anyway to demonstrate and test those workflows on behalf of the rest of the Beman libraries?
I guess yes.